core/src/main/scripts/migrate_db.py

```diff
@@ -0,0 +1,204 @@
+#! /usr/bin/env python
+
```

Comment:
@jjgao a global static final variable would do the job. I like the idea of the message on the web page.